docs/webhooks/index.html

```diff
@@ -253,6 +254,36 @@ <h2>Events for CodeSignal Test</h2>
  <div class="subsection">
 <h3 id="testworkflow">Test webhooks process flow</h3>
 <img alt="Diagram of webhooks process flow for CodeSignal Test" width="100%" src="https://codesignal.com/wp-content/uploads/2021/03/webhooksproces_test.png" />
+</div>
+<div class="subsection">
+    <h3 id="companyTestSessionCreated">companyTestSessionCreated</h3>
+    <p>
+        Fired when a client creates a test session.
+    </p>
+    <div>Request format:</div>
+    <pre><code>{
+event: 'companyTestSessionCreated',
+triggeredOn: number,
+payload: {
+testSessionId: string,
+testId: string,
+testTitle: string,
+candidateEmail: string,
+candidateName: string
+}
+};</code></pre>
+    <div>Sample request:</div>
+    <pre><code>{
+event: 'companyTestSessionCreated',
+triggeredOn: 1553720789347,
+payload: {
+testSessionId: 'lehu382hdleh29',
+testId: 'dhey29hcl28hl28',
+testTitle: 'Software Engineer intern',
+candidateEmail: 'john.doe@gmail.com',
+candidateName: 'John Doe'
+}
+};</code></pre>
```

Comment:
Could you make sure the JSON has proper indentations? This doesn't look great atm:
![image](https://user-images.githubusercontent.com/8866457/218681259-ddb6a8cb-453e-4164-9352-92a301ed559a.png)